_posts/2023-04-18-searchable-snapshots.md

```diff
@@ -0,0 +1,139 @@
+---
+
+The OpenSearch Project is excited to announce the general availability of searchable snapshots in OpenSearch. With searchable snapshots, users can search data segments within snapshots in remote repositories without first having to restore the index data to local storage. The relevant index data is retrieved on-demand with the search request. Searchable snapshots is similar to the UltraWarm storage cluster behavior, using the OpenSearch cluster itself, and employs caching the data on local storage to improve the performance. For further information, see the [UltraWarm Storage documentation](https://docs.aws.amazon.com/opensearch-service/latest/developerguide/ultrawarm.html). In this blog post, we will cover the design and implementation of searchable snapshots, performance, and future enhancements. 
+
+Before searchable snapshots, users had to restore remote snapshots to the OpenSearch cluster node in order to search the data in the snapshots. This required users to size up their infrastructure with larger OpenSearch nodes. 
+
+The searchable snapshot feature incorporates techniques like caching frequently used data segments in cluster nodes and removing the least used data segment from the cluster nodes to make space for frequently used data segments. The data segments downloaded from snapshots on block storage reside alongside the general indexes of the cluster nodes. As such, the computing capacity of cluster nodes is shared between indexing, local search, and data segments on a snapshot residing on lower-cost object storage like Amazon Simple Storage Service (Amazon S3). While cluster node resources are utilized much more efficiently, the high number of tasks results in slower and longer snapshot searches. The local storage of the node is also used for caching the snapshot data. As such, users can search snapshots stored in a lower-cost storage like Amazon S3 with acceptable performance.
+
+The following image demonstrates the creation of an index using a remote storage option, such as Amazon S3.
+
+![Searchable Snapshots](/assets/media/blog-images/search_snap.png){: .img-fluid}
+
+## Enabling searchable snapshot in OpenSearch
+
+To enable searchable snapshots, reference the [instructions](https://opensearch.org/docs/latest/tuning-your-cluster/availability-and-recovery/snapshots/searchable_snapshot/) in the OpenSearch documentation.
+
+## Using a snapshot for search
+
+OpenSearch cluster nodes will retrieve data from a snapshot and then cache it to the node's local storage. You can then configure the amount of storage available for caching snapshot data, or use the system defaults which are based on predetermined logic. Any subsequent searches on the cached data is then served from the node's local storage, or cache, without the need to download the segment data again from the snapshot, which improves performance.
```

Comment:
In the last sentence, if "cache" is just another name for "the node's local storage", then the commas are fine, but if we are referencing two separate things, then no commas.